v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
```

Comment:
I was thinking about `transform` too. The advantage of putting it below it is that **it is** a type of transform. But then we need to think about the source and destination languages. If I call `https://en.wikipedia.org/api/rest_v1/transform/html/language/...`, then is `en` the source or the destination language? Obviously, because we already have the domain, we don't need two languages, but which of these do we not include?

It's not an easy answer. This is why for this round I opted to have a separate `/translate` hierarchy under the global domain as a first step. I do think we (eventually) need to integrate content translation into the proper domains, but it needs careful consideration.